### PR TITLE
Update login.html

### PR DESCRIPTION
**Pull request by schw1erig**

### CODE REVIEW

File: src/main/resources/static/unauthenticated/css/LoginPage.css [hunk 1]
```diff
@@ -79,33 +77,3 @@ img {
     margin: 1% auto auto auto;
 }
 
-@media only screen and (max-width: 768px) {
```
Comment (DarkressX) [inline src/main/resources/static/unauthenticated/css/LoginPage.css:82]: Why is this removed? Do we no longer want to support mobile devices?

File: src/main/resources/templates/login.html [hunk 2]
```diff
@@ -8,21 +8,24 @@
 <body>
 
     <img src="unauthenticated/img/commonlogo3.png">
+</head>
+<body>
+    <img src="commonlogo3.png">
```
Comment (DarkressX) [inline src/main/resources/templates/login.html:13]: Here two things are happening:
1. The body tag is opened twice
2. The logo is displayed twice.
I suspect, that there is an issue

**Review by DarkressX — CHANGES_REQUESTED** [review]
This is how the site currentyl looks, if I fix the image path of the images

image

File: src/main/resources/static/unauthenticated/css/LoginPage.css (hunk 1 above)
Comment (schw1erig) [reply src/main/resources/static/unauthenticated/css/LoginPage.css:82]: It changes automatically

**Review by DarkressX — APPROVED** [review]

**Pull request merged**